src/Application/Request/CreateUserRequest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Application\Request;
+
+class CreateUserRequest
+{
+    private $userName;
+    private $email;
+
+    public function __construct(string $userName, string $email){
+        $this->userName = $userName;
+        $this->email = $email; 
+        
+    }
+
+    public function getUserName(){
```

Comment:
Here, we can add the type of the return, this way when you see the method you know which type is returning.

Reply:
`public function getUserName():string{`